app/src/renderer/components/staking/PageStaking.vue

```diff
@@ -34,7 +30,15 @@ export default {
     ]
   }),
   computed: {
-    ...mapGetters([`connected`, `delegates`, `filters`])
+    ...mapGetters([`connected`, `delegates`, `filters`, `user`])
+  },
+  mounted() {
```

Comment:
should this be a computed? right now if a user signs in while on this page, it will not update correct?

Reply:
if session would be a page, this should not be a problem